documentation/course/E1.md

```diff
@@ -385,6 +417,11 @@ Reguły zostały podzielone według encji, których dotyczą oraz są uporządko
 - `REG/06/11` - Mandat musi zawierać powód.
 - `REG/06/12` - Mandat musi być wystawiony przez dokładnie jednego biletera.
 - `REG/06/13` - Mandat może karać dokładnie jednego pasażera.
+- `REG/06/14` - Mandat musi posiadać status.
+- `REG/06/15` - Status mandatu może przyjmować wartości: opłacony, nieopłacony, anulowany. 
+- `REG/06/15` - Status mandatu jest domyślnie nieopłacony.
+- `REG/06/16` - Status mandatu zmienia się na opłacony po uregulowaniu opłaty.
+- `REG/06/17` - Status mandatu zmienia się na anulowany po anulowaniu mandatu przez biletera.
```

Comment:
Technically speaking, the enum should be described separately (like `FineReason`), but you would have to change the numbering of all `REG`s declared below.